packages/teleport/src/Databases/AddDatabase/AddDatabase.tsx

```diff
@@ -59,6 +90,10 @@ export default function AddDatabase({
       ? `tsh login --proxy=${host}`
       : `tsh login --proxy=${host} --auth=local --user=${username}`;
 
+  const { id: token } = joinToken;
+  const useGeneratedToken = canCreate && token !== '';
```

Comment:
in React community, `use` is reserved prefix used to name hooks. Also, why cannot `canCreate` be used without token check?